src/Roslyn.Suppressions/Roslyn.Suppressions/SupportedSuppressionBuilder.tt

```diff
@@ -4,7 +4,8 @@
 <#@ assembly name="$(SonarCSharpDllPath)" #>
 <#@ assembly name="$(SonarVBDllPath)" #>
 <#@ assembly name="$(MSCodeAnalysisCommonDllPath)" #>
-<#@ assembly name="$(NuGetPackageRoot)system.collections.immutable\1.5.0\lib\netstandard2.0\System.Collections.Immutable.dll" #>
+<#@ assembly name="$(MSCodeAnalysisWorkspacesDllPath)" #>
+<#@ assembly name="$(SystemCompositionAttributedModelDllPath)" #>
```

Comment:
See the note on line 100 below about why these are required when running from the command line